package.json

```
@@ -8,6 +8,7 @@
   },
   "scripts": {
     "cypress": "CYPRESS_BASE_URL=${CYPRESS_BASE_URL:-http://localhost:8080} cypress run",
+    "test:lint-cypress": "eslint --fix cypress/integration/**",
```

```suggestion
    "test:lint-cypress": "eslint --no-fix --max-warnings 0 cypress/integration/**",
```

Mh, this should be similar to `test:lint` with the `--no-fix` and `--max-warning 0` parameters. Though having a fix:cypress command might make sense indeed.

At some point we probably want to use `vue-cli-service lint` instead of eslint directly and maybe also lint the entire `cypress/` directory and not just the `integration/` subdirectory, but I'm still getting an error there that need some more thought. We can do that later.

---

- good suggestion about the `--no-fix` thing
- yea, I tested the whole `cypress/` directory. There are some errors that need more investigation. Its mostly in the `cypress/plugins` directory.